sktime/networks/cntc.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+"""a combination of CCNN and CLSTM as feature extractors."""
+
+__author__ = ["James-Large, Withington, TonyBagnall", "AurumnPegasus"]
```

Comment:
typos

Reply:
Can you please point them out? is it a grammatical mistake or typo?